rootfs/usr/local/bin/aws-config-setup

```diff
@@ -21,6 +21,27 @@ if [ -z "${AWS_ROOT_ACCOUNT_ID}" ]; then
 	exit 1
 fi
 
+if [ ! -d "/localhost/.awsvault/" ] || [ ! -w "/localhost/.awsvault/" ]; then
+	echo "/localhost/.awsvault/ is not writable"
+	exit 1
+fi
+
+if [ -d "/localhost/.awsvault/keys/" ] && [ ! -w "/localhost/.awsvault/" ]; then
+	echo "/localhost/.awsvault/keys/ is not writable"
+	exit 1
+fi
+
+if [ "${AWS_VAULT_BACKEND}" != "file" ]; then
+	echo "*"
+	echo "* WARNING: AWS_VAULT_BACKEND is \"${AWS_VAULT_BACKEND}\". Geodesic only supports AWS_VAULT_BACKEND=\"file\"."
+	echo "*"
+fi
+
+if [ ! -d "/localhost/.awsvault/" ] || [ ! -w "/localhost/.awsvault/" ]; then
```

Comment:
This seems redundant with https://github.com/cloudposse/geodesic/pull/435/files#diff-fc3933c71539ab95454263b39985b8e2R24